SCRigidScript.py

```diff
@@ -0,0 +1,85 @@
+# Python Script, API Version = V19 Beta
+
+chord = 150
+c2 = chord/2
+
```

Comment:
You may consider some more clear variable names. I think you are getting at leading and trailing edge so it may be good to be verbose so others can understand should they use your code.